lib/terraforming/util.rb

```diff
@@ -1,20 +1,20 @@
 module Terraforming
   module Util
     def apply_template(client, erb)
-      ERB.new(open(template_path(erb)).read, nil, "-").result(binding)
+      ERB.new(open(template_path(erb)).read, nil, '-').result(binding)
     end
 
     def name_from_tag(resource, default_name)
-      name_tag = resource.tags.find { |tag| tag.key == "Name" }
+      name_tag = resource.tags.find { |tag| tag.key == 'Name' }
       name_tag ? name_tag.value : default_name
     end
 
     def normalize_module_name(name)
-      name.gsub(/[^a-zA-Z0-9_-]/, "-")
+      name.gsub(/[^a-zA-Z0-9_-]/, '-')
     end
 
     def template_path(template_name)
-      File.join(File.expand_path(File.dirname(__FILE__)), "template", template_name) << ".erb"
+      File.join(File.expand_path(File.dirname(__FILE__)), 'template', template_name) << '.erb'
```

Comment:
__[rubocop]__
* Line is too long. [94/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)